sormas-e2e-tests/src/test/resources/features/sanity/web/Contacts.feature

```diff
@@ -295,3 +295,38 @@ Feature: Contacts end to end tests
     And I click on SAVE button in Link Event to group form
     And I navigate to the last created through API Event page via URL
     And I check that number of displayed Event Participants is 1
+
+  @issue=SORDEV-5640 @env_main
+  Scenario: Fill an exposure data in Contacts
```

Comment:
can we adjust the name to reflect the logic of the test? it's not a just a simple fill in, because we already have this in place